postgres/src/db.py

```diff
@@ -0,0 +1,22 @@
+from typing import Generator
+
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from src.config import postgres_url
+
+url = postgres_url()
+
+# Create a SQLAlchemy engine
+engine = create_engine(url)
+
+# Create a session factory
+SessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=engine)
+
+
+def get_db() -> Generator:
```

Comment:
check fastapi `lifespan events` to do this more idiomatically. https://fastapi.tiangolo.com/advanced/events/

Reply:
I will have a look but the db session is more or less copied verbatim from FastAPI example. 
https://github.com/tiangolo/fastapi/blob/f7e3559bd5997f831fb9b02bef9c767a50facbc3/docs_src/sql_databases/sql_app/main.py#L15